apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,131 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    test('right typeof parameters', function(){
+      assert.throw(() => Utils.throttle(null), 
+                   Error, 
+                   'func must be a Function');
+      assert.throw(() => Utils.throttle(()=>{}, null),
```

Comment:
Should we enforce spaces around arrow in `.eslintrc` with the [arrow-spacing](http://eslint.org/docs/rules/arrow-spacing) rule ?